e2e_samples/fabric_dataops_sample/docs/adr_naming_convention.md

```diff
@@ -0,0 +1,79 @@
+# Naming convention for Fabric E2E sample
+
+This document describes the naming convention for the Fabric E2E sample. It includes the naming convention for the following:
```

Comment:
convention should be conventions since there are many
(two changes here)

Reply:
Updated